examples/megacli.cpp

```diff
@@ -2402,6 +2413,48 @@ void exec_find(autocomplete::ACState& s)
     }
 }
 
+bool recurse_findemptysubfoldertrees(Node* n, bool movetotrash)
+{
+    if (n->type == FILENODE) return false;
+    std::vector<Node*> emptyfolders;
+    bool empty = true;
+    Node* trash = nodebypath("//bin");
```

Comment:
It may sound too picky, but rootnodes handles are available at `rootnodes` vector. Better to avoid a search by path (already thinking of nodes-on-demand, sorry :) )